index.js

```diff
@@ -43,20 +43,24 @@ function reload(name) {
 }
 
 
+var deletedModules = [];
```

Comment:
When is this array cleared? Could the modules stated in it get "polluted" for later calls of `freshy.unload`?
What if instead of having it in the most outer scope you include it as a parameter of the `unload` function and initialize it empty if it is not present?